### DISCUSSION TITLE
Farsi characters are separated in the column

### DISCUSSION THREAD

**Discussion by DanRahimii** [URUWorks/TeroSubtitler#330] [Q&A]
Farsi characters don't show properly on macOS. Can anyone help me through solving that?
<img width="553" alt="Screenshot 1403-08-30 at 12 50 14" src="https://github.com/user-attachments/assets/6011b2bf-6af6-443a-887b-40fce09eb5ea">

**Comment by chenlung** [comment]
Hi, @DanRahimii!

- Would you be able to supply a small sample for checking, please?
- What OS and version of Tero Subtitler are you using?

Tero has since resumed development, so I'm hoping this can be fixed for you shortly (and you can try a development build to be sure).
